scripts/EN/Harry

```diff
@@ -6560,116 +6560,116 @@ CleanScreen(); // 0x84ec: 00000064
-ShowText((2134)"¥"Your role is over. We don't need you anymore. What do you think you can accomplish by coming here?¥""); // 0x85fa: 0000100c
+ShowText((2134)"¥"Your role is over.¥nW2We don't need you anymore.¥nW2What do you think you can accomplish by coming here?¥""); // 0x85fa: 0000100c
 HandleInput(); // 0x8600: 00000010
-ShowText((2135)"A smile appears on Kaufmann's face and he reaches into his pocket. ¥"My, aren't we getting cocky? Bet you can't see this and keep your cool.¥""); // 0x8604: 0000100c
+ShowText((2135)"A smile appears on Kaufmann's face and he reaches into his pocket.¥n¥nW2¥"My, W1aren't we getting cocky?¥nW1Bet you can't see this and keep your cool.¥""); // 0x8604: 0000100c
 HandleInput(); // 0x860a: 00000010
 CleanScreen(); // 0x860e: 00000064
 LoadBackground(0xcc); // 0x8612: 00001069
 DrawBackground(); // 0x8618: 00000065
-ShowText((2136)"The item that Kaufmann pulled out of his pocket... It is definitely the small bottle that I found in the gas tank of the motorcycle that was in that garage somewhere."); // 0x861c: 0000100c
+ShowText((2136)"The item that Kaufmann pulled out of his pocket…¥nW2It is definitely the small vial that I found in the gas tank of the motorcycle that was in that garage somewhere."); // 0x861c: 0000100c
 HandleInput(); // 0x8622: 00000010
-ShowText((2137)"¥"Aglaophotis! I thought I got rid of that!¥" ¥"All I had to do was plant it somewhere for you to find. You, all, well, it kept you busy. Ha! You're easy! And there's more where this came from!¥""); // 0x8626: 0000100c
+ShowText((2137)"¥"Aglaophotis! W2I thought I got rid of that!¥"¥n¥nW2¥"All I had to do was plant it somewhere for you to find.¥nW1You, all, well, it kept you busy.¥nW2Ha! You're easy! And there's more where this came from!¥""); // 0x8626: 0000100c
 HandleInput(); // 0x862c: 00000010
 CleanScreen(); // 0x8630: 00000064
 LoadBackground(0x0); // 0x8634: 00001069
 DrawBackground(); // 0x863a: 00000065
-ShowText((2138)"¥"Stop it!!¥" Kaufmann throws the small bottle at Alessa. All four of us watch the bottle fly in a beautiful parabola, our eyes fixed on the sacred liquid within."); // 0x863e: 0000100c
+ShowText((2138)"¥"Stop it!!¥"¥n¥nW2Kaufmann throws the small vial at Alessa.¥nW2All four of us watch the vial fly in a beautiful parabola, our eyes fixed on the sacred liquid within."); // 0x863e: 0000100c
 HandleInput(); // 0x8644: 00000010
-ShowText((2139)"There is tension. Although she is normally calm and composed, that drug is enough to frighten Dahlia. That small bottle hit a part of Alessa's body, shattered into pieces, and soaked Alessa uncomfortably with the red liquid that was inside."); // 0x8648: 0000100c
+ShowText((2139)"There is tension.¥nW2Although she is normally calm and composed, that drug is enough to frighten Dahlia.¥nW2The small vial superbly strikes part of Alessa's body W1and shatters, W1uncomfortably showering her with the red liquid held within and staining part of her white clothing red."); // 0x8648: 0000100c
```

Comment:
Changed bottle to vial to match PS1 and then modified the entire last sentence to better capture the meaning from the Japanese.